package.json

```diff
@@ -1,23 +1,12 @@
 {
-  "name": "eslint-config-mavenlint",
-  "version": "2.2.2",
-  "description": "Mavenlink ESLint config",
-  "main": "index.js",
-  "repository": "https://github.com/mavenlink/mavenlint.git",
-  "license": "MIT",
-  "devDependencies": {
-    "eslint": "^4.14.0",
-    "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.5.1"
-  },
-  "peerDependencies": {
-    "eslint": ">=4.14.0",
-    "eslint-plugin-import": ">=2.8.0",
-    "eslint-plugin-jsx-a11y": ">= 5.1.1",
-    "eslint-plugin-react": ">=7.5.1"
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "publish-packages": "./node_modules/lerna/bin/lerna.js publish"
```

Comment:
Did the relative path here instead of the global `lerna` command so that people wouldn't have to globally install lerna to do this.

Reply:
In `scripts`, you have access to any local node module. This can just be `lerna publish`.

Reply:
Nice!

Reply:
This should be cleaned up to conform to standard node script practices (accessing the module locally)

Reply:
Wat, this did get cleaned up